string[] name
bool[] save_current_settings
int8[] control_strategy
hebiros/PidGainsMsg position_gains
hebiros/PidGainsMsg velocity_gains
hebiros/PidGainsMsg effort_gains

